wheel/chia_rs.pyi

```diff
@@ -345,6 +345,41 @@ class SpendBundleConditions:
         removal_amount: Union[ int, _Unspec] = _Unspec(),
         addition_amount: Union[ int, _Unspec] = _Unspec()) -> SpendBundleConditions: ...
 
+class MerkleSet:
+    nodes: List[uint64]
+    leafs: List[bytes]
+    hash_cache: List[bytes]
+    def get_root(self) -> bytes: ...
+    def is_included_already_hashed(self, to_check: bytes) -> Tuple[bool, bytes]: ...
+    def check_proof(proof: bytes) -> MerkleSet: ...
+    def __init__(
+        self,
+        nodes: Sequence[uint64],
+        leafs: Sequence[bytes],
+        hash_cache: Sequence[bytes]
+    ) -> None: ...
+    def __hash__(self) -> int: ...
+    def __repr__(self) -> str: ...
+    def __richcmp__(self) -> Any: ...
+    def __deepcopy__(self) -> MerkleSet: ...
+    def __copy__(self) -> MerkleSet: ...
+    @staticmethod
+    def from_bytes(bytes) -> MerkleSet: ...
+    @staticmethod
+    def from_bytes_unchecked(bytes) -> MerkleSet: ...
+    @staticmethod
+    def parse_rust(ReadableBuffer, bool = False) -> Tuple[MerkleSet, int]: ...
+    def to_bytes(self) -> bytes: ...
+    def __bytes__(self) -> bytes: ...
+    def stream_to_bytes(self) -> bytes: ...
+    def get_hash(self) -> bytes32: ...
+    def to_json_dict(self) -> Dict[str, Any]: ...
+    @staticmethod
+    def from_json_dict(json_dict: Dict[str, Any]) -> MerkleSet: ...
+    def replace(self, *, nodes: Union[ List[uint64], _Unspec] = _Unspec(),
+        leafs: Union[ List[bytes], _Unspec] = _Unspec(),
+        hash_cache: Union[ List[bytes], _Unspec] = _Unspec()) -> MerkleSet: ...
```

Comment:
I don't think this type-stub is correct given your python binding.

Reply:
These were autogenerated from `print_class` and then copied. What seems to be wrong specifically?

Reply:
`print_class()` assumes a rust `struct` that derives from `Streamable` and `PyStreamable`. That's where a lot of these member functions come from. I don't think your class should implement these functions, so the simplest thing would be to just hard code the type stub in the generation script